File: Kedar/WPT/Day 3/Factorial.js



// do{
    

// }while(ch != 0);


// function fact(){
    
//     fact = 1;
//     for(i = 1;i<=num;i++){
//         fact = fact * i;
//     }
//     return fact;
// }


function fact(n){
    if(n == 0 || n == 1){
        return 1;
    }
    else{
        return n*fact(n-1)
    }
}
rv = fact(3);
console.log("Factorial of the  is:"+rv);

File: Java/Day 1/Factorial.java
import java.util.Scanner;

class Factorial{


    public int fact(int num){
        if(num <= 0){
            return 1;
        }
        else{
            int f = 1;
            for(int  i= 1;i<=num;i++){
                f =f*i;
            }
            return f;
        }
    }


    public static void main(String args[]){

        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Number : -");
        int num = sc.nextInt();
        Factorial fc = new Factorial();


        int res = fc.fact(num);
        System.out.println("Factorial : -" + res);

    }
}